components/pipeline-service/production/base/update-tekton-config-performance.yaml

```diff
@@ -31,10 +31,23 @@
   path: /spec/pipeline/options/deployments/tekton-operator-proxy-webhook/spec/replicas
   # default pipeline-service setting is 1
   value: 2
-- op: replace
-  path: /spec/pipeline/options/deployments/tekton-pipelines-remote-resolvers/spec/replicas
-  # default pipeline-service setting is 1
-  value: 2
+- op: add
+  path: /spec/pipeline/options/deployments/tekton-pipelines-remote-resolvers/spec
+  # replicas - default pipeline-service setting is 1
+  # memory limit - default upstream setting is 4Gi
+  value:
+    replicas: 2
```

Comment:
ooh ... didn't know you could patch this way with the `op: add` gitops approcah ... cool